volatility3/framework/symbols/linux/extensions/__init__.py

```diff
@@ -1504,39 +1500,40 @@ def is_valid(self):
         )
 
     def _is_kernel_prior_to_struct_mount(self) -> bool:
-        """Helper to distinguish between kernels prior to version 3.3.8 that
-        lacked the 'mount' structure and later versions that have it.
+        """Helper to distinguish between kernels prior to version 3.3 which lacked the
+        'mount' struct, versus later versions that include it.
+        See 7d6fec45a5131918b51dcd76da52f2ec86a85be6.
 
-        The 'mnt_parent' member was moved from struct 'vfsmount' to struct
-        'mount' when the latter was introduced.
+        # Following that commit, also in kernel version 3.3 (3376f34fff5be9954fd9a9c4fd68f4a0a36d480e),
+        # the 'mnt_parent' member was relocated from the 'vfsmount' struct to the newly
+        # introduced 'mount' struct.
 
         Alternatively, vmlinux.has_type('mount') can be used here but it is faster.
 
         Returns:
-            bool: 'True' if the kernel
+            'True' if the kernel lacks the 'mount' struct, typically indicating kernel < 3.3.
         """
 
         return self.has_member("mnt_parent")
 
     def is_equal(self, vfsmount_ptr) -> bool:
         """Helper to make sure it is comparing two pointers to 'vfsmount'.
 
-        Depending on the kernel version, the calling object (self) could be
-        a 'vfsmount \\*' (<3.3.8) or a 'vfsmount' (>=3.3.8). This way we trust
-        in the framework "auto" dereferencing ability to assure that when we
-        reach this point 'self' will be a 'vfsmount' already and self.vol.offset
-        a 'vfsmount \\*' and not a 'vfsmount \\*\\*'. The argument must be a 'vfsmount \\*'.
+        Depending on the kernel version, see 3376f34fff5be9954fd9a9c4fd68f4a0a36d480e,
+        the calling object (self) could be a 'vfsmount *' (<3.3) or a 'vfsmount' (>=3.3).
+        This way we trust in the framework "auto" dereferencing ability to assure that
+        when we reach this point 'self' will be a 'vfsmount' already and self.vol.offset
+        a 'vfsmount *' and not a 'vfsmount **'. The argument must be a 'vfsmount *'.
```

Comment:
The reason for the `\\` should be to escape the * and make sure it prints properly.  If that was overkill, that's fine, but I'd like confirmation someone has tested this and that the stars do actually come out in the compiled documentation as stars and not start-bold-emphasis or something strange...

Reply:
No worries, the escaping is back. That said, I'm not a big fan of massacre the Python docstring just to make it render correctly in HTML. Ideally, it should work the other way around, the module converting the docstring to HTML should handle escaping as needed for the final format. For example, a \\* might render correctly in HTML but could be the wrong escaping for other formats

Reply:
I also tried to test it, but the documentation is currently broken. To fix this, we need the excellent work from https://github.com/volatilityfoundation/volatility3/pull/1548 by @dgmcdona to restore functionality.